packages/grid/src/ux-responsive-utilities.ts

```diff
@@ -1,48 +1,58 @@
 import { PLATFORM } from 'aurelia-pal';
+import { Disposable } from 'aurelia-binding';
 
 /** Utility class that assists with designing a responsive site. */
-export class UxResponsiveUtilities {
+export class UxResponsiveUtilities implements Disposable  {
+
+  /** Current screen height that can be used to set custom breakpoints. */
+  public height: number;
+
+  /** Current screen width that can be used to set custom breakpoints. */
   public width: number;
 
-  constructor() {
-    this.width = PLATFORM.global.innerWidth;
+  /** Visible on screens smaller than 480px. */
+  public xs: boolean = false;
 
-    let recentlyUpdated = false;
+  /** Visible on screens larger than 480px, and smaller than 960px. */
+  public sm: boolean = false;
 
-    window.addEventListener('resize', () => {
-      if (recentlyUpdated) {
-        return;
-      }
+  /** Visible on screens larger than 960px, and smaller than 1280px. */
+  public md: boolean = false;
 
-      recentlyUpdated = true;
-      this.width = PLATFORM.global.innerWidth;
+  /** Visible on screens larger than 1280px, and smaller than 1925px. */
+  public lg: boolean = false;
 
-      setTimeout(() => recentlyUpdated = false , 250);
-    });
-  }
+  /** Visible on screens larger than 1925px. */
+  public xl: boolean = false;
 
-  /** Visible on screens smaller than 480px. */
-  public get xs() {
-    return this.width <= 480;
-  }
+  private recentlyUpdated = false;
 
-  /** Visible on screens larger than 480px, and smaller than 960px. */
-  public get sm() {
-    return this.width > 480 && this.width <= 960;
-  }
+  constructor() {
+    window.addEventListener('resize', () => this.calculateResponsiveValues());
 
-  /** Visible on screens larger than 960px, and smaller than 1280px. */
-  public get md() {
-    return this.width > 960 && this.width <= 1280;
+    this.calculateResponsiveValues();
   }
 
-  /** Visible on screens larger than 1280px, and smaller than 1925px. */
-  public get lg() {
-    return this.width > 1280 && this.width <= 1925;
+  private calculateResponsiveValues() {
+    if (this.recentlyUpdated) {
+      return;
+    }
+
+    this.recentlyUpdated = true;
+
+    this.height = PLATFORM.global.innerHeight;
+    this.width = PLATFORM.global.innerWidth;
+
+    this.xs = this.width <= 480;
+    this.sm = this.width > 480 && this.width <= 960;
+    this.md = this.width > 960 && this.width <= 1280;
+    this.lg = this.width > 1280 && this.width <= 1925;
+    this.xl = this.width > 1925;
+
+    setTimeout(() => this.recentlyUpdated = false, 250);
```

Comment:
Why not polyfill RAF instead? Why 250 throttle instead of a 100 second debounce?